lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -18,6 +18,7 @@ class AwsCompileScheduledEvents {
 
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'schedule', {
       anyOf: [
+        { $ref: '#/definitions/awsCfIf' },
```

Comment:
To avoid ambiguity let's not support CF functions at top level but just at `schedule.rate` field (and allow them at array item level). We also should there support any CF functions not just `If`

Reply:
Makes sense; changed